### DISCUSSION TITLE
Applying policies

### DISCUSSION THREAD

**Discussion by ciscocollabguru** [DependencyTrack/dependency-track#4077] [Q&A]
Looking for guidance on how to actually apply policies to my projects.  I haven't been able to find any info other than policies exist and can be used...appreciate any help you all can provide.

**Comment by chrikoch** [comment]
I'm stuck with the same problem. Does anyone have some hints?
I have for example this policy:

![grafik](https://github.com/user-attachments/assets/0061fde8-96ce-4c94-a458-712571f66f09)

And I definitely have a lot of components, that don't have that license. However, the number of policy violations is still zero. I've waited for >24h and also issued a metrics refresh manually.

**Reply by chrikoch** [reply]
I think I was just too impatient and probably changed the policy while waiting :-( I now understand the policy checks for what you do NOT want to have in your components and the above policy now triggers as expected. So if (policy evaluates to true) then viloation is triggerd.
